providers/implementations/include/prov/names.h

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * Macros for use in our providers.  On place to maintain.
```

Comment:
No idea what 'On place to maintain' means.

Reply:
It means that we don't have this information spread and duplicated all over the place.

Reply:
I removed it

Reply:
I think  s/On/One/ was the confusing part.

Reply:
Ah, right